catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -329,6 +330,9 @@ private AttributeValidator getValidator(Outer.Validator validator) {
             }
             return new RangeValidator(min, max);
         }
+        case "country": {
```

Comment:
How about calling this `iso3_country` or something similar? We might want to add other types of country codes in the future.